doc/2/api/essentials/error-codes/plugin/index.md

```diff
@@ -108,7 +108,7 @@ description: error codes definitions
 | Id | Error Type (Status Code)             | Message           |
 | ------ | -----------------| ------------------ | ------------------ |
 | plugin.context.missing_collection<br/><pre>0x04060001</pre> | [PluginImplementationError](/core/2/api/essentials/error-handling#pluginimplementationerror) <pre>(500)</pre> | The 'collection' argument is required, but none was provided |
-| plugin.context.unavailable_realtime<br/><pre>0x04060002</pre> | [PluginImplementationError](/core/2/api/essentials/error-handling#pluginimplementationerror) <pre>(500)</pre> | Due to technical limitations, plugins have limited access to the realtime API controller |
+| plugin.context.unavailable_realtime<br/><pre>0x04060002</pre> | [PluginImplementationError](/core/2/api/essentials/error-handling#pluginimplementationerror) <pre>(500)</pre> | Due to technical limitations, subscribe and unsubscribe method are only available in the embedded SDK. |
```

Comment:
```suggestion
| plugin.context.unavailable_realtime<br/><pre>0x04060002</pre> | [PluginImplementationError](/core/2/api/essentials/error-handling#pluginimplementationerror) <pre>(500)</pre> | Due to technical limitations, subscribe and unsubscribe methods are only available in the embedded SDK. |
```

Reply:
I changed it in`lib/kerror/codes/plugin.json`